lib/components/ClinicUserEdit.js

```diff
@@ -157,72 +113,33 @@ var ClinicUserEdit = React.createClass({
     }
   },
 
-  renderMonth: function() {
-    var options = _.map(MONTHS, function(item) {
-      return <option key={item.value} value={item.value}>{item.label}</option>;
-    });
-    return (
-      <select
-        className={styles.monthInput}
-        name="month"
-        ref="month"
-        value={this.state.month}
-        onChange={this.handleChange}
-        onBlur={this.handleBlur}>
-        {options}
-      </select>
-    );
-  },
-
-  renderDay: function() {
-    return <input
-      className={styles.dateInput}
-      name="day"
-      ref="day"
-      placeholder="Day"
-      value={this.state.day}
-      onChange={this.handleChange}
-      onBlur={this.handleBlur} />;
-  },
-
-  renderYear: function() {
-    return <input
-      className={styles.dateInput}
-      name="year"
-      ref="year"
-      placeholder="Year"
-      value={this.state.year}
-      onChange={this.handleChange}
-      onBlur={this.handleBlur} />;
-  },
-
-  renderButton: function() {
-    var text = 'Save';
-    return (
-      <button type="submit"
-        className={styles.button}
-        onClick={this.handleNext}
-        disabled={(!!this.state.errorMessage || !this.state.formsFilled)}>
-        {text}
-      </button>
-    );
-  },
-
   renderError: function() {
-    if (this.props.errorDismissed || (!this.props.errorMessage && !_.get(this,['state','errorMessage']))) {
+    if (this.props.errorDismissed || (!this.props.errorMessage )) {
       return null;
     }
     return (
       <div className={styles.error}>
         <span>
-          {this.props.errorMessage}{this.state.errorMessage}<i className={styles.iconClose} onClick={this.props.dismissCreateCustodialAccountError}></i>
+          {this.props.errorMessage}<i className={styles.iconClose} onClick={this.props.dismissCreateCustodialAccountError}></i>
         </span>
       </div>
     );
   },
 
+  renderDateError: function(){
+    const {fields: {fullName, month, day, year}} = this.props;
```

Comment:
Not using `fullName` here, no need to destructure it out. (My AirBnb trained eye sees such things easily now 😜)